package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "failed-to-load",
+  "version": "0.0.0-alpha.1",
+  "description": "Find script tag that have failed to load before window.load",
+  "tags": [
+    "script",
+    "script tag",
+    "load",
+    "fail",
+    "failed to load",
+    "observability",
+    "monitoring",
+    "🏷"
+  ],
+  "type": "module",
+  "browser": "index.js",
```

Comment:
Redundant.
Also, how about a `main` export with the dist?

Reply:
This package is only good for browser. I can think of a main directing to an empty module to avoid "isomorphic" use breakage WDYT?

```js
module.exports.check = () => []
module.exports.register = () => null
```

Reply:
Added "main" containing stubs for nodejs